tools/testdata-scripts/src/bin/download-repo-sources.rs

```diff
@@ -14,7 +14,7 @@ use std::path::PathBuf;
 use zip::ZipArchive;
 
 include!("../../globs.rs.data");
-include!("../../locales.rs.data");
+include!("../../../../provider/datagen/tests/data/locales.rs.data");
```

Comment:
Thought: it's really annoying the this file moves around so much and not I need to remember the new location, but I agree it seems like it should ship in the crate that uses it